pages/internship.js

```diff
@@ -0,0 +1,116 @@
+import Link from 'next/link';
+import React from 'react';
+
+import Faq from 'react-faq-component';
+import Carousel from 'react-multi-carousel';
+
+import Banner from '../components/Banner';
+import CommitteeInternshipInfo from '../components/Internship/CommitteeInternshipInfo';
+import InternshipTimeline from '../components/Internship/InternshipTimeline';
+import NextSteps from '../components/Internship/NextSteps';
+import Layout from '../components/Layout';
+
+import data from '../data';
+import 'react-multi-carousel/lib/styles.css';
+
+function InternshipPage(){
+    const { committees, internship } = data;
+    const { items, testimonials, QA, FAQStyles, responsive } = internship;
+    return(
+        <Layout>
+            <Banner decorative />
+            <div className="content-section text-center">
+                <h1>Internships</h1>
+                What do you do as an intern? Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+                sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Tristique sollicitudin
+                nibh sit amet commodo nulla. Amet facilisis magna etiam tempor.
+            </div>
+            <CommitteeInternshipInfo
+                committees={committees}
+            />
+            <div className="content-section text-center">
+                <h1>Applications</h1>
+                <InternshipTimeline items={items}/>
+                <div className="application-information">
+                    <div>
+                        {items.map((item, i) => (
+                        <div key={i} className='application-item'>
```

Comment:
Similarly, should this semantically be another tag?

Reply:
not sure which tag I'd use here!

Reply:
`<div>` is the most generic tag. [Here's a quick W3Schools article](https://www.w3schools.com/html/html5_semantic_elements.asp) on some options.